src/sentry/utils/performance_issues/performance_detection.py

```diff
@@ -281,6 +281,13 @@ def get_detection_settings(project_id: int | None = None) -> dict[DetectorType,
             "allowed_span_ops": ["http.client"],
             "detection_enabled": settings["n_plus_one_api_calls_detection_enabled"],
         },
+        DetectorType.EXPERIMENTAL_N_PLUS_ONE_API_CALLS: {
+            "total_duration": settings["n_plus_one_api_calls_total_duration_threshold"],  # ms
+            "concurrency_threshold": 10,  # ms
+            "count": 5,
```

Comment:
Note this doesn't match above, in order to increase fire rate, im lowering the count required to 5, and upping the distance allowed between spans to 10ms